versionist.conf.js

```diff
@@ -0,0 +1 @@
+module.exports = require('versionist/versionist.conf');
```

Comment:
Ah, yes, I hadn't seen that one. I'd leave this here anyways until that issue is resolved.